src/gui/sdlmain.cpp

```diff
@@ -1692,6 +1749,42 @@ Bitu GFX_SetSize(int width,
 		if (rinfo.flags & SDL_RENDERER_ACCELERATED)
 			retFlags |= GFX_HARDWARE;
 
+		// Yoinked from OpenGL path; more weight, the faster it'll crumble
```

Comment:
While comments like these are fun, can you please restrict it to factual information? Can you rewrite this?